### PR TITLE
Add bullseye SDKs to be built for SWA

### PR DESCRIPTION
**Pull request by pauld-msft**
- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.

https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1579874
First step of validating we support all of SWA's cases is to make sure we have the following versions of SDKs for bullseye consumption. I also added ruby, as they require that the Jekyll framework is supported.
```
Node.js 12, 14, 16, 18
.NET Core 3.1, .NET 6, 7
Python 3.8, 3.9
Frontend frameworks / static site generators currently supported (e.g., Jekyll, Hugo)
```
- [x] Tests are included and/or updated for code changes.
- [x] Proper license headers are included in each file.

### CODE REVIEW

File: platforms/nodejs/versions/bullseye/versionsToBuild.txt [hunk 1]
```diff
@@ -1,3 +1,12 @@
+12.19.0
+12.20.0
+12.21.0
+12.22.0
+12.22.4
+12.22.6
+12.22.9
+12.22.11
+
```
Comment (pauld-msft) [inline platforms/nodejs/versions/bullseye/versionsToBuild.txt:9]: Note that we do not support Node 18 yet, but @qianz2 is working on that

File: tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs [hunk 2]
```diff
@@ -590,13 +590,23 @@ public void BuildsApplication_ByDynamicallyInstallingSDKs_IntoCustomDynamicInsta
...
+                    DotNetCoreRunTimeVersions.NetCoreApp60,
+                    DotNetCoreSdkVersions.DotNet60SdkVersion,
+                    NetCore6PreviewWebApp,
+                    imageHelper.GetGitHubActionsBuildImage("github-actions-bullseye"));
+                data.Add(
+                    DotNetCoreRunTimeVersions.NetCoreApp70,
+                    DotNetCoreSdkVersions.DotNet70SdkVersion,
+                    NetCore7PreviewMvcApp,
+                    imageHelper.GetGitHubActionsBuildImage("github-actions-bullseye"));
+
```
Comment (pauld-msft) [inline tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs:609]: Validation job should fail for these tests until the Platform binaries job is run to upload to bullseye SDKs to dev.

**Review by cormacpayne — APPROVED** [review]

**Comment by cormacpayne** [comment]
Whoops forgot the most important part: :shipit:

**Comment by pauld-msft** [comment]
Ok, I think that this is good to merge now. Have tested with the following:
[Dotnet](https://github.com/pauld-msft/acr-test/actions/runs/2891512862)
[Python](https://github.com/pauld-msft/acr-test/actions/runs/2891513154)
[Node](https://github.com/pauld-msft/acr-test/actions/runs/2891667450)
[Ruby](https://github.com/pauld-msft/acr-test/actions/runs/2891513437) (note that ruby failed here, and I investigated and added an apt-get dependency and retested the apps locally to ensure that it passed)

**Review by daniv-msft — APPROVED** [review]

**Pull request merged**